specs/capella/beacon-chain.md

```diff
@@ -108,7 +78,7 @@ We define the following Python custom types for type hinting and readability:
 
 ```python
 class Withdrawal(Container):
-    index: WithdrawalIndex
```

Comment:
`index` is useful to discriminate withdrawals at the EL and any extra-protocol infra that tracks them -- can we leave this in?